lib/ts/recipeImplementation.ts

```diff
@@ -308,6 +308,26 @@ export default function RecipeImplementation(recipeImplInput: {
                     return domain === normalisedsessionDomain;
                 }
             }
+        },
+
+        getClockSkewInMillis: function ({
+            accessTokenPayload
+        }: {
+            accessTokenPayload: any;
+            responseHeaders: Headers;
+        }): number {
+            logDebugMessage("getClockSkewInMillis: called");
+
+            const tokenIssuedAt = accessTokenPayload?.iat;
+            if (tokenIssuedAt === undefined || typeof tokenIssuedAt !== "number") {
+                throw new Error("payload iat is undefined or not a number");
```

Comment:
As discussed in the sprint meeting, please update this to only log the fact that the clock skew is not updated because of an unsupported BE sdk version

Reply:
done